### PR TITLE
Update page.tsx

### PR DESCRIPTION
**Pull request by Keyrxng**

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { useState } from "react";
 export default function Dual() {
   const { messages, input, handleInputChange, handleSubmit, data } = useChat();
   const { count, setCount } = useState(0);
-
+  const wawya  = ""
```
Comment (Keyrxng) [inline app/page.tsx:9]: Empty string detected!

Comment (Keyrxng) [inline app/page.tsx:9]: Empty string detected!

Comment (0x4007) [inline app/page.tsx]: This one wasn't caught. To be honest, if you could reverse this behavior that would be the most useful. For example, if new empty strings are introduced in a pull, then annotate them with a warning. If it is merged in, then in future pulls it should be ignored. That would be excellent.

**Comment by 0x4007** [comment]
I'm pretty sure there is a way to annotate only in the files view instead of showing on the conversation view as well like we see here. Can you look into that? Its some type of built in GitHub actions methods I believe. 

As you can see the conversation view is getting cluttered again.

**Comment by Keyrxng** [comment]
> I'm pretty sure there is a way to annotate only in the files view instead of showing on the conversation view as well like we see here. Can you look into that? Its some type of built in GitHub actions methods I believe.
> 
> As you can see the conversation view is getting cluttered again.

I was going to mention that but yeah I'll look into it

**Pull request closed without merging**